app/Commands/Traits/HasDynamicArgs.php

```diff
@@ -2,6 +2,9 @@
 
 namespace App\Commands\Traits;
 
+use Symfony\Component\Console\Input\ArrayInput;
```

Comment:
we no longer need this use, it is not being used.

Reply:
done